src/style.css

```
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.header {
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+  padding: 0 50px;
+}
+
+:root {
+  --header-color: #00acdc;
+}
+
+.image {
+  justify-content: center;
+  display: flex;
+}
+
+.nav-link-text {
+  padding: 23px 0;
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  text-transform: uppercase;
+}
+
+.is-active {
+  color: var(--header-color);
+  padding: 23px 0;
```

redundant since you apply padding to links
```suggestion
  color: var(--header-color);
```